src/hospital/views.py

```diff
@@ -1,3 +1,35 @@
 from django.shortcuts import render
+from django.contrib.auth.models import User
 
+from .models import Patient
+from .forms import PatientForm
+
+import time
 # Create your views here.
+
+def home(request):
+    return render(request, 'index.html')
+
+def patient_form_view(request):
```

Comment:
Is it just a patient form? 
It is better to have separate views for listing patients and other for the creation process.